sections/model.tex

```diff
@@ -0,0 +1,139 @@
+under comparison (in this case: days). Both of these characteristics can aid
+understanding and explanation. In simple terms, the distance measures the
+approximate `minimal work' required to move between two probability
+distributions where `work' can be loosely defined as the product of how much of
+the distribution's mass is to moved and the distance it must be moved
+by. More formally, the Wasserstein distance between two probability
+distributions \(U\) and \(V\) is defined as:
+\begin{equation}\label{eq:wasserstein}
+    W(U, V) = \int_{0}^{1} \left\vert F^{-1}(t) - G^{-1}(t) \right\vert dt
+\end{equation}
+
+\noindent where \(F\) and \(G\) are the cumulative density functions of \(U\)
+and \(V\) respectively. A proof of~\eqref{eq:wasserstein} is presented
+in~\cite{Ramdas2017}. The parameter set with the smallest maximum distance
+between any cluster's simulated system time distribution and the overall
+observed length of stay distribution is then taken to be the most appropriate.
+
+Specifically, the parameter sweep included values of each \(p_i\) from \(0.5\)
+to \(1\) with a granularity of \(10^{-1}\) and values of \(c\) from \(40\) to
+\(60\) at steps of \(5\). These choices were informed by the assumptions of the
+model and formative analysis to reduce the parameter space given the
+computational resources required to conduct the simulations. Each parameter set
+was repeated \(50\) times with each simulation running for four years of virtual
+time. The warm-up and cool-down periods were taken to be approximately one year
+each leaving two years of simulated data from each repetition.
+
+\begin{figure}
+    \centering%
+    \includegraphics[width=\imgwidth]{best_params}
+    \caption{A histogram of the simulated and observed length of stay data for
+             the best parameter set.}\label{fig:best_params}
+\end{figure}
+
+\begin{figure}
+    \centering%
+    \includegraphics[width=\imgwidth]{worst_params}
+    \caption{A histogram of the simulated and observed length of stay data for
+             the worst parameter set.}\label{fig:worst_params}
+\end{figure}
+
+The results of this parameter sweep can be summarised in
+Figures~\ref{fig:best_params}~and~\ref{fig:worst_params}. Each figure shows a
+comparison of the observed lengths of stay across all groups and the newly
+simulated data with the best and worst parameter sets respectively. It can be
+seen that, in the best case, a very close fit has been found. Meanwhile,
+Figure~\ref{fig:worst_params} highlights the importance of good parameter
+estimation under this model since the likelihood of short-stay patient arrivals
+has been inflated disproportionately against the tail of the distribution.
+Table~\ref{tab:comparison} reinforces these results numerically, showing a
+clear fit by the best parameters across the board.
+
+\begin{table}
+    \centering
+    \input{tex/comparison}
+    \caption{A comparison of summary statistics between the observed data, and
```

Comment:
Can we include the Wasserstein distance in the table? (It would be 0 for the observed one).

Reply:
Can we also include the c and the p?

Reply:
That would be sensible. I've included them in 6824457.